ui/src/components/system/Backup.vue

```diff
@@ -3183,6 +3217,150 @@ export default {
     cleanLastLog() {
       this.currentDataBackup.lastLog = null;
     },
+    openCheckDestination(b){
+      var context = this;
+      
+      context.currentDataBackup = context.initBackupData();
+      context.currentDataBackup.name = b.name;
+      context.currentDataBackup.checkDestinationDescription = null;
+      context.currentDataBackup.checkDestinationResult = null;
+      
+      var method = "exec";
+      var api = "system-backup/validate";
+      var configObj = null;
+      
+      switch(b.props.VFSType){
+	    case "cifs":
+	      configObj = {
+		    SMBShare: b.props.SMBShare,
+			SMBHost: b.props.SMBHost,
+			SMBLogin: b.props.SMBLogin,
+			SMBPassword: b.props.SMBPassword,
+			HostToPing: b.props.SMBHost
+	      }
+		  break;
+		  	
+	    case "smb":
+	      configObj = {
+		    NFSHost: b.props.NFSHost,
+		    NFSShare: b.props.NFSShare,
+		    HostToPing: b.props.NFSHost
+	      }
+		break;
+		  	
+	    case "usb":
+	      configObj = {
+	        USBLabel: b.props.USBLabel,
+	        HostToPing: null
+	      }
+		break;
+		  	
+		case "sftp":
+		  configObj = {
+		    SftpHost: b.props.SftpHost,
+		    SftpPort: b.props.SftpPort,
+		    SftpUser: b.props.SftpUser,
+		    SftpDirectory: b.props.SftpDirectory,
+		    HostToPing: b.props.SftpHost
+		  }
+		break;
+		  
+		case "webdav":
+		  configObj = {
+		    WebDAVLogin: b.props.WebDAVLogin,
+		    WebDAVPassword: b.props.WebDAVPassword,
+		    WebDAVUrl: b.props.WebDAVUrl,
+		    HostToPing: 'nethserver.org' /* checking internet connection */
+		  }
+		break;
+		  
+		case "b2":
+		  configObj = {
+            B2AccountId: b.props.B2AccountId,
+		    B2AccountKey: b.props.B2AccountKey,
+		    B2Bucket: b.props.B2Bucket,
+		    HostToPing: 'nethserver.org' /* checking internet connection */
```

Comment:
Why nethserver.org should be related to Backblaze server?
If nethserver.org is down, why Backblaze should be considered down too?